### PR TITLE
added an anvil section 

### PR DESCRIPTION
**Pull request by Perelyn-sama**
If you feel I missed something, comment on it and I'll add it ASAP

### CODE REVIEW

File: src/anvil/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Overview of Anvil
+
+Anvil is Foundry's local testnet node implementation written in Rust. You can use it for testing your contracts from frontends or interacting over RPC.
+
+### How to get Anvil
+
+You can get Anvil on all Foundry-supported platforms with the latest `foundryup`.
+
+1. Update foundryup: `curl -L https://foundry.paradigm.xyz | bash`
+2. Run it: `foundryup`
```
Comment (onbjerg) [inline src/anvil/README.md:10]: We can just mention that it is installed as part of Foundry and link to the installation section

File: src/anvil/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+### How to get Anvil
+
+You can get Anvil on all Foundry-supported platforms with the latest `foundryup`.
+
+1. Update foundryup: `curl -L https://foundry.paradigm.xyz | bash`
+2. Run it: `foundryup`
+
+### How to use Anvil
+
+To use Anvil, run `anvil`
```
Comment (onbjerg) [inline src/anvil/README.md:14]: Missing period

File: src/anvil/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+You should see a list of accounts and private keys available for use, there are also some subcommands you could run to configure the test node. You can run `anvil -h` to see them.
```
Comment (onbjerg) [inline src/anvil/README.md:16]: They are not subcommands, they are options :smile:

File: src/anvil/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Some examples of them are:
+
+```bash
+//  Number of dev accounts to generate and configure. [default: 10]
```
Comment (onbjerg) [inline src/anvil/README.md:21]: Use bash comments in bash. Alternatively, we can remove the examples and link to the Anvil reference (see my other comment)

File: src/anvil/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+anvil -a, --accounts <ACCOUNTS>
+
+// The EVM hardfork to use. [default: latest]
+anvil --hardfork <HARDFORK>
+
+// Port number to listen on. [default: 8545]
+anvil  -p, --port <PORT>
+```
+
+### Examples
```
Comment (onbjerg) [inline src/anvil/README.md:31]: Unneeded; it works like any other node

**Review by onbjerg — COMMENTED** [review]
Thanks! We should add an Anvil command reference, just like we have for Forge and Cast. Additionally, you need to link to this file in `src/SUMMARY.md`, otherwise it won't show up anywhere.

In that command reference the content described in the original issue should be present, e.g. mining modes, what JSONRPC calls are supported etc.

File: src/anvil/README.md (hunk 5 above)
Comment (Perelyn-sama) [reply src/anvil/README.md:31]: Okay man, thanks. I just saw this right now. 

I'll get working.

**Pull request merged**